mkdocs.yml

```diff
@@ -69,7 +71,11 @@ pages:
       - "Commit Logs": "m3db/architecture/commitlogs.md"
       - "Peer Streaming": "m3db/architecture/peer_streaming.md"
       - "Caching": "m3db/architecture/caching.md"
-  - "Query Engine and Coordinator":
+  - "Coordinator":
+    - "Introduction": "coordinator/index.md"
+    - "API":
+      - "Prometheus Remote Read/Write": "coordinator/api/remote.md"
```

Comment:
Doesn't the `remote.md` document only explain how to write, not read?

Reply:
Correct I'll add the docs on how to read.